x-pack/plugins/session_view/public/components/session_view_detail_panel/helpers.ts

```diff
@@ -20,10 +21,13 @@ const getDetailPanelProcessLeader = (leader: ProcessFields) => ({
 
 export const getDetailPanelProcess = (process: Process) => {
   const processData = {} as DetailPanelProcess;
+  const endProcessList = process.events.filter((items) => items.event.action === 'end');
 
   processData.id = process.id;
   processData.start = process.events[0]['@timestamp'];
-  processData.end = process.events[process.events.length - 1]['@timestamp'];
+  processData.end = dataOrDash(
```

Comment:
if you pull the latest, i believe the `dataOrDash` is applied in `detail_panel_process_tab` already when rendering. https://github.com/elastic/kibana/blob/main/x-pack/plugins/session_view/public/components/detail_panel_process_tab/index.tsx#L316

you won't need to include `dataOrDash` here. We usually just want to use `dataOrDash` just before rendering the data since its a fallback, and we want to avoid mistaking the dash as the actual data along the way.